fe/src/components/WalletButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
+    class="relative"
+    @mouseover="isDropdownOpen = true"
+    @mouseleave="isDropdownOpen = false"
+  >
+    <button
+      v-if="props.connected"
+      class="flex gap-4 justify-center items-center btn-gradient"
+    >
+      <Icon name="solar:wallet-linear" size="25" />
+      <span>{{ props.address }}</span>
```

Comment:
After destructuring the props, we no longer need to add the prefix `props.`
```suggestion
      <span>{{ address }}</span>
```

Reply:
Apply to related.